application/src/main/java/org/thingsboard/server/controller/AuthController.java

```diff
@@ -128,7 +125,7 @@ public ObjectNode changePassword(
 
             sendEntityNotificationMsg(getTenantId(), userCredentials.getUserId(), EdgeEventActionType.CREDENTIALS_UPDATED);
 
-            eventPublisher.publishEvent(new UserAuthDataChangedEvent(securityUser.getId()));
+            eventPublisher.publishEvent(new UserAuthDataChangedEvent(securityUser.getId(), securityUser.getSessionId(), false));
```

Comment:
Please divide UserAuthDataChangedEvent into multiple: e.g. UserCredentialsInvalidationEvent and UserSessionInvalidationEvent